rpc/connector/connector.proto

```diff
@@ -196,13 +196,19 @@ message IngressInfos {
   repeated manager.IngressInfo ingress_infos = 1;
 }
 
-message InstallRequest {
+message HelmRequest {
+  enum Type {
+    UNSPECIFIED = 0;
+    INSTALL = 1;
+    UPGRADE = 2;
+    UNINSTALL = 3;
+  }
   ConnectRequest connect_request = 1;
-  bool upgrade = 2;
+  Type type = 2;
   repeated string value_paths = 3;
 }
 
-message InstallResult {
+message Result {
```

Comment:
nit can this be `HelmResult`

Reply:
Not really, because it's also used by the `Uninstall` method.